src/module.api/loan.controller.ts

```diff
@@ -174,6 +183,50 @@ export class LoanController {
     return await this.vaultService.get(id)
   }
 
+  /**
+   * List vault auction history.
+   *
+   * @param {string} id vaultId
+   * @param {number} height liquidation height
+   * @param {number} batchIndex batch index
+   * @param {PaginationQuery} query
+   * @return {Promise<ApiPagedResponse<VaultAuctionBatchHistory>>}
+   */
+  @Get('/vaults/:id/auctions/:height/history/:batchIndex')
```

Comment:
```suggestion
  @Get('/vaults/:id/auctions/:height/batches/:batchIndex/history')
```

1 more correction should be this to be more explicit.